package.json

```diff
@@ -46,14 +46,14 @@
     "@guardian/eslint-plugin-source-foundations": "^4.0.2",
     "@guardian/eslint-plugin-source-react-components": "^4.0.2",
     "@guardian/node-riffraff-artifact": "^0.3.2",
-    "@storybook/addon-actions": "^6.5.0-alpha.34",
-    "@storybook/addon-essentials": "^6.5.0-alpha.34",
-    "@storybook/addon-links": "^6.5.0-alpha.34",
-    "@storybook/builder-webpack5": "^6.5.0-alpha.34",
-    "@storybook/manager-webpack5": "^6.5.0-alpha.34",
-    "@storybook/react": "^6.5.0-alpha.34",
+    "@storybook/addon-actions": "^6.5.0-alpha.40",
+    "@storybook/addon-essentials": "^6.5.0-alpha.40",
+    "@storybook/addon-links": "^6.5.0-alpha.40",
+    "@storybook/builder-webpack5": "^6.5.0-alpha.40",
+    "@storybook/manager-webpack5": "^6.5.0-alpha.40",
+    "@storybook/react": "^6.5.0-alpha.40",
```

Comment:
Do we think it's a good idea to bump prerelease versions unless there's a specific change we need? I would suggest we leave these alone until the next proper semver release.

Reply:
agreed, will revert